packages/api-specification/transform-type-info.js

```diff
@@ -62,8 +62,15 @@ if (program.testfile) {
     const testReport = JSON.parse(fs.readFileSync(program.testfile));
     Object.keys(testReport).forEach((testMethod) => {
       const getSelectionString = (type, name, childName) => `..{(.kindString === "Class" || .kindString === "Interface") && .name === "${name}"}.children{.kindString === "${type}"` + (childName ? `&& .name === "${childName}"}` : '}');
+      const getPassed = obj => obj ? obj.passed : 0;
+      const getTotal = obj => obj ? obj.total : 0;
```

Comment:
It looks like this may introduce a variation on #273 if we don't have a set of results? Do we want to be showing an empty test run in this case?

Reply:
Do you mean where Browser doesn't implement the feature, like this?

![getbounds-test-open](https://user-images.githubusercontent.com/8255295/30902842-b181bb2a-a364-11e7-8d6f-a5d81f30b08e.png)

Reply:
I think the problem wouldn't occur because it only calculates the `combined` score if there's a `results` object in the `testReport` for that method.

There are a couple of odd ones where it reports `Electron 0/0 OpenFin 0/0`, but that was already happening before (e.g. Window.reload). The test-report.json file actually has that data in it.

Reply:
That's exactly the problem, but you're right that it's now removed from the test results before this runs. :+1: